index.js

```diff
@@ -55,6 +55,9 @@ function fastifyRedis (fastify, options, next) {
       }
 
       fastify.decorate('redis', client)
+      if (options.closeClient === true) {
+        fastify.addHook('onClose', close)
+      }
```

Comment:
I think we would need to support when this is `false` in all cases.

Reply:
I'm not sure about that. The issue is around passing in a specific client and managing that outside instance.

Reply:
I mean, if somebody is not passing a client in and `closeClient: false`, then _closing_ the client seems off.

Reply:
I'm unclear why they would do that or why it would even be supported. They wouldn't be able to manage it without first pulling the instance from Fastify.